tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -144,6 +160,9 @@ def _test_fsx_lustre(
     storage_type=None,
     auto_import_policy=None,
     deployment_type=None,
+    weekly_maintenance_start_time=None,
+    imported_file_chunk_size=None,
+    storage_capacity=None,
```

Comment:
This `_test_fsx_lustre` function is getting very complicated and these default parameters are getting a bit out of hand here :sweat_smile:

IMO we should extract the common assertions also used by other test into a separate function like `_assert_basic_fsx_functions` and rename this function to something like `_assert_fsx_configuration_options` and get rid of all the default `=None`

The issue I see with `=None` here is we don't really need them here, and having them here will not produce an obvious failure if we decreased test coverage, because no error will be shown if we remove any of these parameters by accident

This way you can also get rid of all the conditions within your assertion functions like `if storage_capacity:`

The bottom line is we should always be enforcing our assertions strictly in our tests. If things change, we would want to see an error rather than not being aware of the issue.

Reply:
Done